.circleci/docker/common/install_rocm.sh

```diff
@@ -117,23 +127,46 @@ install_centos() {
       echo "gpgkey=http://repo.radeon.com/rocm/rocm.gpg.key" >> /etc/yum.repos.d/amdgpu.repo
   fi
 
-  local rocm_baseurl="http://repo.radeon.com/rocm/yum/${ROCM_VERSION}"
+  if [[ $OS_VERSION == 9 ]]; then
+      local rocm_baseurl="http://compute-artifactory.amd.com/artifactory/list/rocm-osdb-centos-9/compute-rocm-dkms-no-npi-hipclang-10378"
```

Comment:
As stated in offline discussions, we might want to refer to an env variable here to allow the user to provide the repo path for CentOS Stream 9, instead of specifying the non-public repo path?

Reply:
I am thinking setting as 
`local rocm_baseurl="invalid-url"` should be ok for now. 
Currently made this change `centos_stream9_rocm5.3_internal_testing` branch it works for DevOps scripts.

Reply:
@rraminen can you please make the above suggested change?

Reply:
done